src/DataTable/BulkActions.jsx

```diff
@@ -24,9 +25,10 @@ const CustomToggle = React.forwardRef(({ children, onClick }, ref) => (
 ));
 
 const BulkActions = ({
-  actions, selectedRows, className, ...rest
+  actions, className, tableName, ...rest
 }) => {
   const { width } = useWindowSize();
+  const { selectedFlatRows, rows } = useContext(TableContext).getTableInstance(tableName);
```

Comment:
Why afford for multiple tables in a table context?

Reply:
Heh, I think because of an off-hand comment that @adamstankiewicz made once. And I kind of think we'll want tables that interact with each other eventually and it's not hard to do?
Potentially also allows you to maintain table state even if you navigate away from the page and comes back?

Reply:
The idea that a table context contains two or more tables seem unintuitive and I think you could save table state or something in either case.